core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.ts

```diff
@@ -0,0 +1,374 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS O = null KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component for the top navigation bar. This excludes the part
+ * of the navbar that is used for local navigation (such as the various tabs in
+ * the editor pages).
+ */
+
+import { Subscription } from 'rxjs';
+import { Component, Input, OnDestroy, OnInit } from '@angular/core';
+import { ClassroomBackendApiService } from 'domain/classroom/classroom-backend-api.service';
+import { SidebarStatusService } from 'services/sidebar-status.service';
+import { UrlInterpolationService } from 'domain/utilities/url-interpolation.service';
+import { DebouncerService } from 'services/debouncer.service';
+import { SiteAnalyticsService } from 'services/site-analytics.service';
+import { UserService } from 'services/user.service';
+import { DeviceInfoService } from 'services/contextual/device-info.service';
+import { WindowDimensionsService } from 'services/contextual/window-dimensions.service';
+import { SearchService } from 'services/search.service';
+import { NavigationService } from 'services/navigation.service';
+import { AppConstants } from 'app.constants';
+import { I18nLanguageCodeService } from 'services/i18n-language-code.service';
+import { WindowRef } from 'services/contextual/window-ref.service';
+import { downgradeComponent } from '@angular/upgrade/static';
+import { UserBackendApiService } from 'services/user-backend-api.service';
+import { FocusManagerService } from 'services/stateful/focus-manager.service';
+
+interface EventToCodes {
+  enter?,
+  tab?,
+  shiftTab?,
+}
```

Comment:
change this to
```typescript
interface EventToCodes {
  enter?: number;
  tab?: number;
  shiftTab?: number;
}
```
Notice the semicolon instead of `,`

Reply:
Can't use `number` these should be ideally of type `KeyboardEvent` .Not Sure here so left it like this